api/v2/controllers/channels/channels_controller.py

```diff
@@ -1,10 +1,67 @@
 from flask import make_response, jsonify
-from flask_restful import Resource
-from api.v2.helpers.channels.channels_helper import query_all_channels
+from flask_restful import Resource, request
+from api.v2.helpers.channels.channels_helper import (query_all_channels,
+                                                     validate_calendar_data)
+from api.v2.models.channels.channels_model import Channels as channels_model
+from api.v2.helpers.bouquets.bouquets_helper import get_bouquet
+from googleapiclient.errors import HttpError
+from api.v2.helpers.credentials import check_bouquet_credentials
+
+import os
+import uuid
 
 
 class Channels(Resource):
     def get(self):
         channels = query_all_channels()
+        return make_response(jsonify({'channels': channels}), 200)
+
+    def post(self):
+        # Enpoint to add channels to a bouquet
+        try:
+            calendar = request.get_json()
+
+            # Function call to validate channel data
+            is_valid = validate_calendar_data(calendar)
+            if is_valid.errors:
+                return {'error': is_valid.errors}, 400
+
+            # Function call to retrieve a bouquet
+            bouquet = get_bouquet(calendar['bouquet_id'])
```

Comment:
I think this function of 8 lines can be refactored to `bouquet = Bouquets.query.filter_by(var=var).first()`

Reply:
Using that method will not allow me to access the bouquet dictionary that I need. which is why I used that function `get_bouquet` to get and convert the result into a python dictionary